docker/nwchem/Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM        ubuntu:14.04
+
+LABEL maintainer="Muammar El khatib <melkhatibr@lbl.gov>, Chris Harris <chris.harris@kitware.com>"
+
+ENV         LC_ALL=C.UTF-8
+ENV         LANG=C.UTF-8
+ENV         NWCHEM_TOP="/opt/nwchem"
+
+RUN         apt-get update \
+            && apt-get -y upgrade \
+            && apt-get install -y gfortran libopenblas-dev libopenmpi-dev \
+                                  openmpi-bin tcsh make ssh patch curl git
```

Comment:
Looks like we are running nwchem in serial mode, do we need openmpi?

Reply:
I changed this here https://github.com/OpenChemistry/mongochemdeploy/pull/51/commits/492ae4eefe53c7eba852b3be285e35ec134e4795 thanks for noticing that. This `cpus` should be passed in the `context` dictionary. Right now is just hardcoded. Is that ok to add a `cpus` entry in the input? Is that the right property name?